app/views/admin/cms/_collection_list.html.erb

```diff
@@ -0,0 +1,41 @@
+<table>
+  <thead>
+    <tr>
+      <th>Collection ID</th>
+      <th>Title</th>
+      <th>Items</th>
+      <th>Published</th>
+      <th>Actions</th>
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody id="collection_list">
+    <% Collection.all.each do |resource| %>
```

Comment:
interesting that you call the variable `resource`,  most people would call it `collection` (just a side comment, not a request to change)